en/mkdocs.yml

```diff
@@ -75,7 +75,8 @@ nav:
   - Introduction:
       - What is Choreo?: what-is-choreo.md
   - Quick Start Guide:
-      - Build Your First Cloud Native Application with Choreo: quick-start-guides/build-your-first-cloud-native-application-with-choreo.md
+      - Deploy Your First Service: quick-start-guides/deploy-your-first-service.md
+      - Build Your First Cloud Native Full Stack Application: quick-start-guides/build-your-first-cloud-native-full-stack-application.md
```

Comment:
Are we going with this title or with "Deploy a Web Application that Consumes a Backend Service"?

Reply:
yes we can do that